ci/helm/preview/values.yaml

```diff
@@ -0,0 +1,93 @@
+expose:
+  Annotations:
+    helm.sh/hook: post-install,post-upgrade
+    helm.sh/hook-delete-policy: hook-succeeded
+  config:
+    # add tls configuration
+    domain: platform.dev.nuxeo.com
+    exposer: Ingress
+    http: true
+    tlsacme: true
+    tlsSecretName: platform-cluster-tls
+    urltemplate: '"{{.Service}}-{{.Namespace}}.{{.Domain}}"'
+
+cleanup:
+  Args:
+    - --cleanup
+  Annotations:
+    helm.sh/hook: pre-delete
+    helm.sh/hook-delete-policy: hook-succeeded
+
+nuxeo:
+  fullnameOverride: preview
+  nuxeo:
+    customParams: |-
+      elasticsearch.restClient.socketTimeoutMs=300000
+      elasticsearch.restClient.connectionTimeoutMs=300000
+    podAnnotations:
+      preview/scm.ref: ${SCM_REF}
+    podLabels:
+      branch: "$BRANCH_NAME"
+      team: platform
+      resource: pod
+      usage: preview
+    service:
+      annotations:
+        fabric8.io/ingress.annotations: |-
+          nginx.ingress.kubernetes.io/proxy-body-size: 0
+          nginx.ingress.kubernetes.io/server-snippet: ignore_invalid_headers off;
+      labels:
+        team: platform
+        resource: service
+        usage: preview
+    image:
+      repository: ${DOCKER_REGISTRY}/nuxeo/nuxeo
+      tag: ${VERSION}
+      pullPolicy: Always
```

Comment:
`VERSION` should refer to a release version, so always unique, thus `pullPolicy: Always` should be useles.

Reply:
since this is the default value of the nuxeo helm chart we need to override it with `IfNotPresent` or `Never`
I think `IfNotPresent` would be best

Reply:
Yes indeed.

In, fact, while we're relying on the `latest` tag in the nuxeo Helm chart, we could remove `pullPolicy: Always` from it, see https://kubernetes.io/fr/docs/concepts/containers/images/#mettre-%C3%A0-jour-des-images.

But we might want to use something else than `latest` in the future, since it is not recommended.

Reply:
Thanks for the extra infos!